can/logger.py

```diff
@@ -185,6 +185,17 @@ def main() -> None:
         default=None,
     )
 
+    parser.add_argument(
+        "-t",
+        "--file_time",
+        dest="file_time",
+        type=int,
+        help="Maximum period in seconds before rotating log file. (If file_size"
+        "is also given, then the first of the two constraints to occur is"
+        "what causes the rollover.)",
```

Comment:
Is this right?

```suggestion
        help="Maximum period in seconds before rotating log file. (If file_size "
        "is also given, then the first of the two constraints to occur is "
        "what causes the rollover.)",
```

Reply:
Yes, this is true of the current implementation. The one exception being that if there is no CAN messages sent on the bus, then the `rollover` condition is not checked for. Therefore, if the CAN bus is inactive and the time limit is passed, a rollover will only occur after a message is received. At first I thought this was problematic, but I'm not sure if there is a benefit of creating empty log files. However, if that is a desired feature, then something would need to change.

Reply:
This is also how the TimedRotatingLogger is implemented. 
![image](https://user-images.githubusercontent.com/39248734/209858048-2f9d7087-8cab-4c62-bbf9-f415f73ffe86.png)

Reply:
Ah sorry, I meant to ask whether my suggestion to add spaces is right. I could've been clearer. 😅

But thank you for your explanation. I also think that the current implementation is fine in that regard!